JeffreyQ/UCLA-CS-130
# Jq/create poll db

Major refactor to the frontend create poll page. Broke out poll creation pages into separate files. 


### ### TESTING 
Be sure to clear out AsyncStorage so that you can authenticate through Facebook as part of the test. 

1.Spin up the frontend on an ios simulator
2. Navigate to the Create Poll Page.
3. Select any poll and input strings into the fields before pressing "Create".
4. Verify that the poll is successfully created, via React Native Debugger, or querying the ec2 instance endpoints to view all polls and users.
5. Repeat for all other poll creation options
